### PR TITLE
Add remote servers to server using env variable

### PR DESCRIPTION
**Pull request by EduPonz**
eProsima/Fast-DDS#1767 enables configuring remote servers for a server using env variable. This PR adds a test for the functionality.

### CODE REVIEW

File: test/configuration/test_cases/test_34_connect_locally_with_remote_server.xml [hunk 1]
```diff
@@ -1,185 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (jparisu) [inline test/configuration/test_cases/test_34_connect_locally_with_remote_server.xml:1]: Why is this file erased?

File: test/configuration/tests_params.json [hunk 2]
```diff
@@ -1238,7 +1238,56 @@
...
+                    }
+                },
+                "fastddstool_1":
+                {
+                    "kill_time": 5,
+                    "tool_config":
+                    {
+                        "id" : 0,
+                        "address": "127.0.0.1",
+                        "port": 34811
```
Comment (jparisu) [inline test/configuration/tests_params.json:1252]:
```suggestion
                        "port": 32811
```
Test 32, use 32k port instead.

**Review by jparisu — CHANGES_REQUESTED** [review]
Please, check that erased test case should really be deleted

File: test/configuration/test_cases/test_34_connect_locally_with_remote_server.xml (hunk 1 above)
Comment (EduPonz) [reply test/configuration/test_cases/test_34_connect_locally_with_remote_server.xml]: This has been put back in 4d558d0

File: test/configuration/tests_params.json (hunk 2 above)
Comment (EduPonz) [reply test/configuration/tests_params.json]: Test renamed to 35 in 4d558d0

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**